Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -121,17 +121,20 @@ private void RenderFloorQuad()
                         UnityEngine.Experimental.XR.Boundary.Type.TrackedArea))
                     {
                         Debug.Log("Got dimensions of tracked area.");
-                        if (dimensions != null)
+                        if (dimensions != Vector3.zero)
                         {
-                            Debug.Log("Drawing floor at dimensions Y.");
-                            // Draw the floor at boundary Y.
+                            Debug.Log("Drawing floor at height offset: " + dimensions.y);
                             floorQuadInstance.transform.localPosition = new Vector3(0, dimensions.y, 0);
                         }
+                        else
+                        {
+                            Debug.Log("Drawing floor at 0,0,0.");
```

Comment:
This if/else is basically "if dimensions is Vector3.zero, draw the floor at Vector3.zero". If dimensions can never be null, we can probably remove the check entirely and just draw at Vector3(0, dimensions.y, 0)